projects/storefrontlib/src/cms-components/misc/icon/icon.module.ts

```diff
@@ -1,13 +1,18 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { provideDefaultConfig } from '@spartacus/core';
+import { defaultIconConfig } from './default-icon.config';
 import { fontawesomeIconConfig } from './fontawesome-icon.config';
 import { IconComponent } from './icon.component';
 
 @NgModule({
   declarations: [IconComponent],
   imports: [CommonModule],
-  providers: [provideDefaultConfig(fontawesomeIconConfig)],
+  providers: [
+    provideDefaultConfig(defaultIconConfig),
+    // TODO: move the opinionated fontawesome config to a recipe
```

Comment:
👍 Which recipe do you mean? Do we plan to copy it both to b2c recipe and b2b recipe? I'm happy to have it here, until we have a clear answer.

---

The need for LTR/RTL flipping for a specific icon type depends on the actual icon resource (how it looks like). What do you think about encapsulating `fontawesomeIconConfig` and default icon fliping config in a new `FontAwesomeIconsModule`. In 3.0 we could move it to `StorefrontModule` (so it's easier to opt-out).

Reply:
yes, it font-awesome is quite opinionated and I think it should be imported in the b2c vs b2b recipes. I like to postpone it for now. Regardless of the icon lib, css driven, svg driven, I actually expect the same icons to be flipped. But you have a point that it could change per lib.

Reply:
I'll keep it as-is for now.